torch/_dynamo/symbolic_convert.py

```diff
@@ -1858,8 +1859,11 @@ def __init__(
                 self._freevars_ids[name] = id(f_locals[name])
 
     def run(self):
-        _step_logger()(logging.INFO, f"torchdynamo start tracing {self.f_code.co_name}")
-        super().run()
+        with TracingContext.current_frame(self.frame_summary()):
+            _step_logger()(
+                logging.INFO, f"torchdynamo start tracing {self.f_code.co_name}"
+            )
```

Comment:
This looks like callee patches though?

Reply:
Callee is preferred, if that means within the instruction translator. I would prefer to keep it an internal detail.